StripeIdentity/StripeIdentity/Source/NativeComponents/Views/DocumentWarmupView.swift

```diff
@@ -51,63 +49,49 @@ class DocumentWarmupView: UIView {
         return label
     }()
 
-    private let documentWarmupBodyLabel: UILabel = {
-        let label = UILabel()
-        label.numberOfLines = 0
-        label.textAlignment = .center
-        label.font = IdentityUI.instructionsFont
-        label.adjustsFontForContentSizeCategory = true
-        label.text = String.Localized.documentFrontWarmupBody
-        label.textColor = IdentityUI.secondaryLabelColor
-        return label
-    }()
-
-    private let accepedIdContainer: UIView = {
-        let uiView = UIView()
-        uiView.layer.borderWidth = 1.0
-        uiView.layer.borderColor = IdentityUI.separatorColor.cgColor
-        uiView.layer.cornerRadius = 12
-        return uiView
-    }()
-
     private let acceptedIds: UILabel = {
         let label = UILabel()
         label.numberOfLines = 0
+        label.textAlignment = .center
         label.font = IdentityUI.instructionsFont
         label.adjustsFontForContentSizeCategory = true
-        label.textColor = IdentityUI.secondaryLabelColor
+        label.textColor = IdentityUI.textColor
         return label
     }()
 
     init(staticContent: StripeAPI.VerificationPageStaticContentDocumentSelectPage) {
         super.init(frame: .zero)
         installViews()
-        bindAcceptTypesOfId(allowList: Array(staticContent.idDocumentTypeAllowlist.values))
+        bindAcceptTypesOfId(allowList: Array(staticContent.idDocumentTypeAllowlist.keys.map { value in
+            if value == "driving_license" {
+                return String.Localized.driverLicense
+            } else if value == "id_card" {
+                return String.Localized.governmentIssuedId
+            } else if value == "passport" {
+                return String.Localized.passport
+            } else {
+                return ""
+            }
```

Comment:
If we do return "", seems like these need to be filtered out?

Reply:
thanks for the pointer! just replaced with `compatMap`